webui/react/src/services/api.ts

```diff
@@ -27,13 +28,21 @@ export const isLoginFailure = (e: any): boolean => {
   return e.response && e.response.status && e.response.status === 403;
 };
 
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+export const isNotFound = (e: any): boolean => {
```

Comment:
question: for future use?

Reply:
nah seems like I forgot to use it. https://github.com/determined-ai/determined/pull/795/files#diff-afa4dcb145f465ed696ff6ee10e2781bR29 we would check the error to see if it's actually a 404 or some other error.

Reply:
follow up to that would be https://determinedai.atlassian.net/browse/DET-3500?filter=-2

Reply:
added that logic. (ss below)